OpenRA.Mods.Common/Traits/SupportPowers/AirstrikePower.cs

```diff
@@ -19,12 +18,25 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public class AirstrikePowerSquadMember
+	{
+		public readonly string UnitType;
+
+		public readonly WVec SpawnOffset;
+
+		public readonly WVec TargetOffset;
+
+		public AirstrikePowerSquadMember(MiniYamlNode yamlNode)
+		{
+			FieldLoader.Load(this, yamlNode.Value);
+		}
+	}
+
 	public class AirstrikePowerInfo : SupportPowerInfo
 	{
-		[ActorReference(typeof(AircraftInfo))]
-		public readonly string UnitType = "badr.bomber";
-		public readonly int SquadSize = 1;
-		public readonly WVec SquadOffset = new WVec(-1536, 1536, 0);
+		[FieldLoader.LoadUsing("LoadSquad")]
+		[Desc("A list of aircraft in the squad. Each has configurable UnitType, SpawnOffset and TargetOffset.")]
```

Comment:
- `nameof(LoadSquad)`
- Does `nameof(AirstrikePowerSquadMember.UnitType)`, etc work and do the right thing here?

Reply:
It should, yes. But note that this was written 2 years ago when we were still on .NET 4.6.1 probably 😆 
Will update.

Reply:
> Feature 'constant interpolated strings' is not available in C# 7.3. Please use language version 10.0 or greater. [OpenRA.Mods.Common]csharp(CS8370)

Well I guess we ***can't*** use `nameof` with `DescAttribute` yet.